sentry-kotlin-multiplatform/src/appleMain/kotlin/io/sentry/kotlin/multiplatform/SentryApple.kt

```diff
@@ -29,6 +34,9 @@ internal actual object SentryBridge {
         val sentryAppleOptions = SentryOptions()
         sentryAppleOptions.dsn = options.dsn
         sentryAppleOptions.attachStacktrace = options.attachStackTrace
+        sentryAppleOptions.beforeSend = { event ->
+            dropKotlinCrashEvent(event as SentryNSExceptionEvent?) as SentryEvent?
```

Comment:
I don't follow why we need this: `dropKotlinCrashEvent`

Reply:
The SDK automatically captures one crash event on iOS, namely SIGABRT which will be stored as well. We also create a custom event that has the proper stack trace (the primary purpose for this lib). `dropkotlinCrashEvent` is responsible for not sending the [SIGABRT](https://sentry.io/organizations/sentry-sdks/issues/3452869503/?project=5903800&query=is%3Aunresolved) to sentry.io. So without that 2 crash events would be sent if a crash happened, unless it's better that way, what do you think?